.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["es2015", "react"]
+  "presets": ["es2015", "react", "stage-0"]
```

Comment:
Sorry that's for Spread operator used [here](https://github.com/code-mancers/rbkit-electron-redux/pull/4/files#diff-58ed19aaca2d7ef6489531b4b3594890R21).